docs/user-event/install.mdx

```diff
@@ -0,0 +1,36 @@
+---
+id: install
+title: Installation
+---
+
+import Tabs from '@theme/Tabs'
+import TabItem from '@theme/TabItem'
+
+<Tabs defaultValue="npm" values={[
+    { label: 'npm', value: 'npm' },
+    { label: 'Yarn', value: 'yarn' },
+]}>
+<TabItem value="npm">
+
+```sh
+npm install --save-dev @testing-library/user-event@^14.0.0-beta
```

Comment:
Ah, never mind v14 is "still" in beta.
I was under the impression that v14 was already released because of "I'd say user-event@14 is a very mature tool that I would recommend anywhere."

Reply:
I added a blockquote note in the `Introduction`.